config/README.md

```diff
@@ -25,3 +25,18 @@ the default:
 ```shell
 ko apply -f config/default.yaml
 ```
+
+### GCP PubSub
```

Comment:
can you add documentation for `gcppubsub-source-key` and the assumptions on the service account?

I think this does not work out of the box for gke for example, you have to grant rights to the default account.